### PR TITLE
examples: closer goto, and go up, not down

### PR DESCRIPTION
**Pull request by julianoes**
This is required because the goto altitude is absolute.

### CODE REVIEW

File: examples/goto.py [hunk 1]
```diff
@@ -28,7 +28,10 @@ async def run():
 
     await asyncio.sleep(1)
 
-    await drone.action.goto_location(55.8688660, -4.2851267, 40, 0)
+    await drone.action.goto_location(47.399386, 8.535245, 500, 0)
+
+    while True:
```
Comment (JonasVautherin) [inline examples/goto.py:33]: Why is that necessary? :thinking:

**Comment by iwishiwasaneagle** [comment]
@julianoes does this work with `plane`?

File: examples/goto.py [hunk 2]
```diff
@@ -28,7 +28,10 @@ async def run():
 
     await asyncio.sleep(1)
 
-    await drone.action.goto_location(55.8688660, -4.2851267, 40, 0)
+    await drone.action.goto_location(47.399386, 8.535245, 500, 0)
```
Comment (iwishiwasaneagle) [inline examples/goto.py:31]: Change heading angle to `float('nan')` such that `await drone.action.goto_location(47.399386, 8.535245, 500,float('nan')`

File: examples/goto.py (hunk 1 above)
Comment (julianoes) [reply examples/goto.py:33]: Oops that was just me debugging.

File: examples/goto.py (hunk 2 above)
Comment (julianoes) [reply examples/goto.py:31]: Oh yes, that would be nicer.

**Comment by julianoes** [comment]
@iwishiwasaneagle do you know why the plane crashes? Is it just because it's too low for it do to crazy turns?

**Comment by iwishiwasaneagle** [comment]
> @iwishiwasaneagle do you know why the plane crashes? Is it just because it's too low for it do to crazy turns?

Well the docs for (MAV_CMD_DO_REPOSITION)[https://mavlink.io/en/messages/common.html#MAV_CMD_DO_REPOSITION] say that for planes, yaw angle should either be 0 or 1 for loiter direction. However, as soon as I do either of those my SITL Plane loses complete control.  

 > Yaw heading. NaN to use the current system yaw heading mode (e.g. yaw towards next waypoint, yaw to home, etc.). For planes indicates loiter direction (0: clockwise, 1: counter clockwise)
